docs/src/main/tut/client.md

```diff
@@ -84,14 +83,14 @@ import cats._, cats.effect._, cats.implicits._
 import org.http4s.Uri
 import scala.concurrent.ExecutionContext.Implicits.global
 
-def hello(name: String): IO[String] = {
+def hello(name: String): fs2.Stream[IO, String] = {
   val target = Uri.uri("http://localhost:8080/hello/") / name
-  httpClient.expect[String](target)
+  httpClient.evalMap(_.expect[String](target))
 }
 
-val people = Vector("Michael", "Jessica", "Ashley", "Christopher")
+val people = fs2.Stream("Michael", "Jessica", "Ashley", "Christopher").covary[IO]
 
-val greetingList = fs2.async.parallelTraverse(people)(hello)
+val greetingList = people.flatMap(hello).compile.toVector
```

Comment:
we might want to leverage `concurrently` here to preserve the parallel nature of the original example, what do people think?